Update console.rst: reversing constants (logical purpose)

ajie62 opened the pull request:
For logical purpose, I added a `!` right before `$this->requirePassword` in the `configure()` method, otherwise the requirePassword argument would be set as REQUIRED instead of OPTIONAL.

<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/roadmap for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `master` for features of unreleased versions).

-->

ajie62 committed:
Update console.rst

For logical purpose, I added a `!` right before `$this->requirePassword` in the `configure()` method, otherwise the requirePassword argument would be set as REQUIRED instead of OPTIONAL.

apfelbox commented:
It would increase the readability, if we just reversed the two constants, don't you think?

ajie62 commented:
Yes, it would! Anyway, I just wanted to point out the logical issue here. Reversing the two constants would do the job! :)

ajie62 committed:
Removed "!", reversed constants instead

Removed the "!" and just reversed the constants as suggested by Jannik Zschiesche

javiereguiluz commented:
@ajie62 nice first contribution to Symfony Docs! Thanks and congrats on it. We merged it in 3.4 (the oldest maintained branch that contains the error) and we'll merge in the other branches automatically later.
